database/src/pool/postgres.rs

```diff
@@ -1413,6 +1414,121 @@ where
             .await
             .unwrap();
     }
+
+    async fn get_benchmark_requests_by_status(
+        &self,
+        statuses: &[BenchmarkRequestStatus],
+        days: Option<i32>,
```

Comment:
I don't think we have any use-case for the `days` parameter anymore? We should be loading all completed (and all pending) requests from the DB, regardless of their age.

Reply:
Removed; 88a431cb8a591ebca195481a1985b58f01b2f628